Update setup.sh

dfredell opened the pull request:
Add a few checks to the setup

dfredell committed:
Update setup.sh

Add a few checks to the setup

ehlertjd commented:
I'll have to give this a shot on a fresh install - I'm not using (I think) vundle anymore

dfredell commented:
I added Vundle because the install instruction for https://github.com/ycm-core/YouCompleteMe#macos calls for it, but you are right. The setup.sh is now just using git clone to install YouCompleteMe

dfredell committed:
Update setup.sh

Remove Vundle install. YouCompleteMe is installing via git clone.

dfredell committed:
Update setup.sh

Install srcery.vim

dfredell committed:
Update setup.sh

Copy scripts dir, make swapfiles dir

dfredell committed:
Update setup.sh

I guess I have some leaning to do with apt-vim. PlugInstall installs YouCompleteMe

dfredell committed:
Add check for vim, copy vimrc
